packages/gutenpride-template/README.md

```diff
@@ -0,0 +1,13 @@
+# Gutenpride Template
+
+A package containing a block template to use with @wordpress/create-block.
```

Comment:
It would be great to add a note that it's used in the official tutorial and link to it.

Reply:
Added a note.